gtsam/nonlinear/tests/testValues.cpp

```diff
@@ -589,6 +589,23 @@ TEST(Values, MatrixDynamicInsertFixedRead) {
   CHECK_EXCEPTION(values.at<Matrix23>(key1), exception);
 }
 
+TEST(Values, Demangle) {
+  Values values;
+  Matrix13 v; v << 5.0, 6.0, 7.0;
+  values.insert(key1, v);
+  string expected = "Values with 1 values:\nValue v1: (Eigen::Matrix<double, 1, 3, 1, 1, 3>) [\n	5, 6, 7\n]\n\n";
```

Comment:
I'm not sure relying on "\n" being universally captured from cout as "\r" vs "\r\n", etc. 
But let's see if it passes all tests on CI for Linux/Mac/Windows !

Reply:
That's an excellent point. I guess I've been living in Linux land for far too long.